hepcrawl/spiders/desy_spider.py

```diff
@@ -281,7 +281,7 @@ def _get_marcxml_records(response_body):
     def _hep_records_from_marcxml(self, marcxml_records):
         def _create_json_record(xml_record):
             try:
-                object_record = create_record(etree.XML(xml_record))
+                object_record = create_record(xml_record)
```

Comment:
you don't need `create_record`. `marcxml2record` takes a string containing XML and spits out, if all goes well, a JSON record.